### PR TITLE
Create mocking function, add fetchTimeEntries test

### PR DESCRIPTION
**Pull request by saravdberg**

### CODE REVIEW

File: services/time-entry-api.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { TimeEntryInterface } from "../fixtures/time-entries";
 import { NotFoundError } from "./not-found-error";
 
 export async function getTimeEntries(): Promise<TimeEntryInterface[]> {
-  return fetch(`${process.env.NEXT_PUBLIC_DB_HOST}/time-entries?_sort=startTime&_order=asc`, {
+  return fetch(`${process.env.NEXT_PUBLIC_DB_HOST}/time-entries`, {
```
Comment (MauriceMarkslag) [inline services/time-entry-api.ts:5]: Why did you delete the query params?

**Review by MauriceMarkslag — APPROVED** [review]

**Review by Doenja — APPROVED** [review]
